.github/workflows/daily.yml

```diff
@@ -1220,7 +1220,7 @@ jobs:
     if: |
       (github.event_name == 'workflow_dispatch' ||
         (github.event_name == 'schedule' && github.repository == 'valkey-io/valkey') ||
-        (github.event_name == 'pull_request' && github.event.pull_request.base.ref != 'unstable')) &&
+        (github.event_name == 'pull_request' && (contains(github.event.pull_request.labels.*.name, 'run-extra-tests') || github.event.pull_request.base.ref != 'unstable'))) &&
```

Comment:
@enjoy-binbin I was going to revert this change before merging. Sorry i didn't mention it.

I guess it's fine to run schema validator in run-extra-tests, but it takes one hour to run.